readthedocs/search/faceted_search.py

```diff
@@ -322,14 +318,7 @@ def query(self, search, query):
             path='sections',
             fields=self._section_fields,
         )
-
-        domains_nested_query = self._get_nested_query(
-            query=query,
-            path='domains',
-            fields=self._domain_fields,
-        )
-
-        queries.extend([sections_nested_query, domains_nested_query])
+        queries.append(sections_nested_query)
```

Comment:
Shouldn't this be `query.extend()` here? In the old code it seems we were extending the list instead of appending a new item.

Reply:
The previous code was using extend because there were two items, not it's just one, so we don't need to use extend.